csharp/test/Ice/proxy/AllTests.cs

```diff
@@ -116,6 +120,7 @@ public static IMyClassPrx Run(TestHelper helper)
                 "ice+ws://host.zeroc.com//identity?protocol=ice1", // invalid protocol
                 "ice+tcp://host.zeroc.com/identity?alt-endpoint=host2?protocol=ice2", // protocol option in alt-endpoint
                 "ice+tcp://host.zeroc.com/identity?foo=bar", // unknown option
+                "ice+tcp://host.zeroc.com/identity?invocation-timeout=0s", // 0 is not a valid invocation timeout
```

Comment:
We should also add a test for `prx.Clone(invocationTimeout: TimeSpan.Zero)`.

Reply:
added a test for this